pkg/log/log_file.go

```diff
@@ -55,9 +56,21 @@ var (
 	logDir        = ""
 
 	executableName = filepath.Base(os.Args[0])
+
+	// Klogv2Flags is the flag set for klog (klogv2).
+	Klogv2Flags flag.FlagSet
+
+	logFlushFreq = logFlushFreqDefault
 )
 
+func init() {
+	klog.InitFlags(&Klogv2Flags)
+}
+
 func AddFlags(fs *pflag.FlagSet) {
+	fs.AddGoFlagSet(&Klogv2Flags)
+	// mimics InitLogs in "k8s.io/component-base/logs"
+	fs.DurationVar(&logFlushFreq, logFlushFreqFlag, logFlushFreqDefault, "Maximum number of seconds between log flushes")
```

Comment:
Do we need to change the value, or just use the fixed 5 seconds?

Reply:
The current Antrea code exposes this parameter but I don't believe it is honored, because we call `InitLogs` (https://github.com/kubernetes/component-base/blob/v0.18.4/logs/logs.go#L58) before parsing the command-line args.